packages/medusa/src/controllers/products/admin-list-products.ts

```diff
@@ -16,13 +16,18 @@ type ListContext = {
   allowedFields?: string[]
   defaultFields?: (keyof Product)[]
   defaultRelations?: string[]
+  include_discount_prices?: boolean
 }
 
 const listAndCount = async (
   scope: AwilixContainer,
   query: FilterableProductProps,
   body?: object,
-  context: ListContext = { limit: 50, offset: 0 }
+  context: ListContext = {
+    limit: 50,
+    offset: 0,
+    include_discount_prices: false,
+  }
 ): Promise<AdminProductsListRes> => {
   const { limit, offset, allowedFields, defaultFields, defaultRelations } =
     context
```

Comment:
out of curiosity, why is there two utils file with `getListConfig` one in `api/routes/admin/tax-rates/utils` and one in `utils`